### ISSUE TITLE
Fix Microsoft XML to JSON 

### ISSUE THREAD

**Issue by adulau** [MISP/misp-warninglists#95]
Fix Microsoft XML to JSON

**Comment by kx499** [comment]
https://github.com/MISP/misp-warninglists/pull/96

Tackled it there, but had some questions about cidr and substring matches in the same list. The old one was set up that way and seemed to work so I'm not sure.

**Comment by adulau** [comment]
Good catch. I separated the generation into 2 warning-lists one for the CIDR block and the other for the string matching. It should be better but maybe some improvements could be done.
